src/newSeed/baseStage.scss

```diff
@@ -185,4 +186,30 @@
       cursor: pointer;
     }
   }
+
+  &.stage6 {
+    & .subtext {
+      width: 30em;
+    }
+    & .linksContainer {
+      width: 30em;
+      display: flex;
+      flex-flow: column;
+      gap: 2em;
+      margin-bottom: 13em;
+      > div {
+          > a {
```

Comment:
Several of these are global defaults for `a` (see styles.scss). Please delete the unnecessary ones here.